RoT/setup-RoT.tp2

```diff
@@ -94,11 +94,13 @@ LANGUAGE ~Espanol~
 		 ~RoT/Language/Spanish/journal.tra~
 		 ~RoT/Language/Spanish/setup.tra~
 
-LANGUAGE ~Simplified Chinese (Optimization and translation By Youye)~
+//Replace the original Optimization and translation by Youye
+LANGUAGE ~Simplified Chinese (retranslation By Lzw104522773)~
 		 ~schinese~
+		 ~RoT/Language/schinese/install.tra~
 		 ~RoT/Language/schinese/scripts.tra~
 		 ~RoT/Language/schinese/journal.tra~
-		 ~RoT/Language/schinese/setup.tra~
+//		 ~RoT/Language/schinese/setup.tra~
```

Comment:
You should include setup.tra file to load all items translations. To keep console prompts in proper encoding, put install.tra file after setup.tra in this list.

Reply:
@Lzw104522773 Please add install.tra as the last file in the list. Now setup.tra will overwrite the prompts from install.tra